compass/landice/tests/greenland/mesh.py

```diff
@@ -56,136 +59,67 @@ def run(self):
 
+        # Create scrip file for the newly generated mesh
         logger.info('creating scrip file for destination mesh')
-        scrip_from_mpas('GIS.nc', 'GIS.scrip.nc')
-        args = ['create_SCRIP_file_from_MPAS_mesh.py',
-                '-m', 'GIS.nc',
-                '-s', 'GIS.scrip.nc']
-        check_call(args, logger=logger)
-
-        # Create weight files from datasets to mesh
-        if exists('BedMachine_to_MPAS_weights.nc'):
-            logger.info('BedMachine_to_MPAS_weights.nc exists; skipping')
-        else:
-            logger.info('generating gridded dataset -> MPAS weights')
-            args = ['srun', '-n', nProcs, 'ESMF_RegridWeightGen', '--source',
-                    data_path + 'BedMachineGreenland-v5.scrip.nc',
-                    '--destination',
-                    'GIS.scrip.nc',
-                    '--weight', 'BedMachine_to_MPAS_weights.nc',
-                    '--method', 'conserve',
-                    "-i", "-64bit_offset",
-                    "--dst_regional", "--src_regional", '--netcdf4']
-            check_call(args, logger=logger)
-
-        if exists('measures_to_MPAS_weights.nc'):
-            logger.info('measures_to_MPAS_weights.nc exists; skipping')
-        else:
-            logger.info('generating gridded dataset -> MPAS weights')
-            args = ['srun', '-n', nProcs, 'ESMF_RegridWeightGen', '--source',
-                    data_path + 'greenland_vel_mosaic500.scrip.nc',
-                    '--destination',
-                    'GIS.scrip.nc',
-                    '--weight', 'measures_to_MPAS_weights.nc',
-                    '--method', 'conserve',
-                    "-i", "-64bit_offset", '--netcdf4',
-                    "--dst_regional", "--src_regional", '--ignore_unmapped']
-            check_call(args, logger=logger)
-
-        # interpoalte fields from BedMachine and Measures
-        # Using conservative remapping
-        logger.info('calling interpolate_to_mpasli_grid.py')
-        args = ['interpolate_to_mpasli_grid.py', '-s',
-                data_path + '/BedMachineGreenland-v5_edits_floodFill_extrap.nc',  # noqa
-                '-d', 'GIS.nc', '-m', 'e',
-                '-w', 'BedMachine_to_MPAS_weights.nc']
-        check_call(args, logger=logger)
-
-        logger.info('calling interpolate_to_mpasli_grid.py')
-        args = ['interpolate_to_mpasli_grid.py', '-s',
-                data_path + '/greenland_vel_mosaic500_extrap.nc',
-                '-d', 'GIS.nc', '-m', 'e',
-                '-w', 'measures_to_MPAS_weights.nc',
-                '-v', 'observedSurfaceVelocityX',
-                'observedSurfaceVelocityY',
-                'observedSurfaceVelocityUncertainty']
-        check_call(args, logger=logger)
-
-        logger.info('Marking domain boundaries dirichlet')
-        args = ['mark_domain_boundaries_dirichlet.py',
-                '-f', 'GIS.nc']
-        check_call(args, logger=logger)
```

Comment:
As far as I can tell `mark_domain_boundaries_dirichlet.py` is called in the `compass.landice.mesh.build_mali_mesh` function, so I've deleted the additional call here. 

Is that OK to do? Or should this be left here?

Reply:
Good catch. Yes, I believe that's redundant. Go ahead and remove.